src/components/Form.jsx

```diff
@@ -41,7 +41,8 @@ export default class Form extends React.Component {
   /* Context which is accepted by Form */
```

Comment:
Here's a list where we would also need to wrap the callback functions:

* utils/fieldUtils/getErrorMessages.js@58
* utils/fieldUtils/validateAsync.js@26
* utils/fieldUtils/validateSync.js@97, 104, 124

I think those are all utils which expose callbacks that can benefit from `withImmutable`.